provision-contest/ansible/generate_passwords.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
```

Comment:
Will this script be called in some Makefile (later)?

Reply:
I did not do that yet, and not sure if we would want that so I prefer to do that outside of this PR.